languages/ruby/ruby_on_rails.md

```diff
@@ -3,34 +3,44 @@
 
-# Like Camping
+# Like Camping ⛺
 When camping there is a rule to leave your campsite cleaner than it was when you got there.  The same is true when writing code.  Each commit should be refactoring the minutia as you go such that each commit leaves the code base cleaner than when you started.
 
 # Documentation
-There are many things that we will need to communicate and share with others developers during and after we’re done.  Please document the basics that help folks know how to install, configure and deploy the app.  We are committed to updating these documents when things change to keep the docs useful.  Given that that’s a hard task, please use documentation wisely.  Too much and it’s unusable and all out of date.  If a dragon hides in the system, write about it.  There are three places that we’ll likely want to send folks: the app’s README, the repo’s Github wiki, and inline code comments. 
+There are many things that we will need to communicate and share with others developers during and after we’re done.  Please document the basics that help folks know how to install, configure and deploy the app.  
+
+We are committed to keeping documentation updated and relevant. The value of documentation diminishes with redundancy. Avoid excessive details that can render the documentation cluttered which increases the chance it will become outdated. Instead, focus on key information and unique system nuances – if there's a 'dragon' lurking in the system, make sure to document it.  Your first point of reference should be the repository's README.md, but the documentation shouldn't stop there. Depending on the organization, a wiki might also be a suitable medium for extended documentation. 
+
+Furthermore, consider the names and structure of your integration tests. Well named tests can act as a form of documentation, providing insights into the systems features, shedding light on our original design intentions, and highlighting edge cases.
 
 ## READMEs
-Please delete the Rails boilerplate one when starting a new app and start plumbing it with the what’s-where basics.  This page is what’s shown on the Github homepage for the project and is a good launching point to other pages on the wiki. Use Markdown when appropriate to improve formatting.
+Please delete the Rails boilerplate one when starting a new app and start plumbing it with the what’s-where basics.  This page is what’s shown on the Github homepage for the project and is a good launching point to other pages in the documentation. Use Markdown when appropriate to improve formatting.  Always aim for a balance between "enough to handle common tasks" and "too long to effectively find what I need".
```

Comment:
Should we be more prescriptive?

At the very least, the page should include project purpose and setup.

Reply:
resolved?